Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -196,20 +195,6 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
         )
     }
 
-    // Sets hashed fxa sync device id for glean deletion ping
-    func setSyncDeviceId() {
-        // Grab our token so we can use the hashed_fxa_uid and clientGUID for deletion-request ping
-        guard let accountManager = RustFirefoxAccounts.shared.accountManager.peek(),
```

Comment:
This was what used to be holding everything together -- as this gets called before `setUpPreLaunchDependencies` which is where the CI crashes. Though shared.accountManager is never initialized at this point, so this never actually worked...